andela/lotus-ah-frontend
# #160451577 Article Report

#### What does this PR do?
A user should be able to report an article

#### Description of Task to be completed?
- implement a functionality that allows a user to report an article
- Build out the report reducers to handle state update

#### How should this be manually tested?
- start the app
- select an article to view
- on the view article page, a report button is available
- click on the button, a modal will appear
- enter the reason for the report and click submit report

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160451577](https://www.pivotaltracker.com/story/show/160451577, "Users should be able to report an article")

#### Screenshots (if appropriate)

<img width="711" alt="screen shot 2018-11-27 at 9 11 28 am" src="https://user-images.githubusercontent.com/20061324/49067722-17b13000-f225-11e8-8c30-e3683d9d747a.png">



#### Questions:
